app/factory/Layer.js

```diff
@@ -567,6 +573,78 @@ Ext.define('CpsiMapview.factory.Layer', {
         Ext.log.info('Not implemented yet', path);
     },
 
+    /**
+     * Creates a XYZ based Vector Tile layer
+     *
+     * @param  {Object} layerConf  The configuration object for this layer
+     * @return {ol.layer.VectorTile}  Vector Tile layer
+     */
+    createVectorTilesLayer: function (layerConf) {
+
+        // transform OL2 properties to current ones supported by OL >=v3
+        var olSourceProps = this.ol2PropsToOlSourceProps(layerConf.openLayers);
+        var olLayerProps = this.ol2PropsToOlLayerProps(layerConf.openLayers);
+
+        var olSourceConf = {
+            format: new ol.format[layerConf.format](),
```

Comment:
Should we check whether this format-class exists prior to instantiating?

Reply:
Done in 90d88b0